pkg/core/pipeline/conditions.go

```diff
@@ -43,20 +44,24 @@ func (p *Pipeline) RunConditions() (globalResult bool, err error) {
 				p.Sources[condition.Config.SourceID].Output +
 				p.Sources[condition.Config.SourceID].Config.Postfix)
 
-		if err != nil {
-			// Show error to end user if any
-			logrus.Error(err)
-			// Fail globally (even though other conditions will be executed)
-			globalResult = false
-		} else {
-			// Update pipeline's configuration after each condition run
+		rpt.Result = condition.Result
+
+		if err == nil {
+			// Update pipeline after each condition run
 			err = p.Config.Update(p)
 			if err != nil {
 				globalResult = false
 				return globalResult, err
 			}
 		}
-		rpt.Result = condition.Result
+
+		if err != nil {
```

Comment:
```suggestion
		if err != nil || strings.Compare(condition.Result, result.SUCCESS) != 0 {	
```

Otherwise the globalresult will be set to true if there is not error but that the condition failed

Reply:
and `logrus.Error(err)` should only be execute if err != nil

Reply:
updated: WDYT?